### PR TITLE
Add flask server

### PR DESCRIPTION
**Pull request by tschaffter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,3 +61,12 @@ Then run :
     2020-08-24 15:22:20,820 root - INFO - Inserted row 1 
     ...
 
+
+### Flask REST Front End
+
+Create the flask app from the https://github.com/data2health/nlp-sandbox-schemas CodeBase
+
+    cd ~project/nlp-sandbox-schemas
```
Comment (tschaffter) [inline README.md:69]: Replace with something like:

1. Download OpenAPI spec from XXX (we will replace with the url later)
2. Install openapi-generator...
3. Generate initial server

Comment (tschaffter) [reply README.md:69]: Maybe add a section about how to update to server implementation is a new version of the OpenAPI spec is available.

File: flask-app/git_push.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```
Comment (tschaffter) [inline flask-app/git_push.sh:1]: Remove this file?

File: flask-app/openapi_server/templates/home.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
Comment (tschaffter) [inline flask-app/openapi_server/templates/home.html:1]: This page may be removed later and we will deny requests sent to the root path.

File: flask-app/openapi_server/test/test_date_controller.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
```
Comment (tschaffter) [inline flask-app/openapi_server/test/test_date_controller.py:1]: Can you add in the README how to run these unit tests?

File: flask-app/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+connexion[swagger-ui] >= 2.6.0; python_version>="3.6"
```
Comment (tschaffter) [inline flask-app/requirements.txt:1]: Why is there two requirements.txt? We should keep only one.

File: flask-app/git_push.sh (hunk 2 above)
Comment (gkowalski) [reply flask-app/git_push.sh:1]: that was a stock generated file . Removed.

File: flask-app/requirements.txt (hunk 5 above)
Comment (gkowalski) [reply flask-app/requirements.txt:1]: one was stock generated for python 3.5 and 3.6 . Does not work for 3.7+ so I added another . If we don't want backwards compatibility I'll remove one.

Comment (tschaffter) [reply flask-app/requirements.txt:1]: Let's keep only one `requirements.txt` for 3.7. Anyone can easily set up a python environment with 3.7.

File: README.md (hunk 1 above)
Comment (tschaffter) [reply README.md:69]: Can you also add a section that describes how to start the stack with `docker-compose`?

Comment (gkowalski) [reply README.md:69]: I've been adding doc to the ~/flask-app/README.md Should I move those to the top level README.md ?

File: flask-app/openapi_server/templates/home.html (hunk 3 above)
Comment (gkowalski) [reply flask-app/openapi_server/templates/home.html:1]: Not sure why ? It provides a quick test for Dev-ops and us to see the container is running and does not  introduce a security risk.

File: README.md (hunk 1 above)
Comment (tschaffter) [reply README.md:69]: This look good for now. Later we will need to section: 1) how to deploy the stack using docker-compose, and 2) how to start the server manually (current section) for development purpose.

File: flask-app/openapi_server/templates/home.html (hunk 3 above)
Comment (tschaffter) [reply flask-app/openapi_server/templates/home.html:1]: It's fine to keep the page but we can make it look nicer in the future, e.g. showing information from `/health`.

File: flask-app/openapi_server/test/test_date_controller.py (hunk 4 above)
Comment (tschaffter) [reply flask-app/openapi_server/test/test_date_controller.py:1]: Unit tests will be added in another PR. Resolving this conversation.

**Comment by tschaffter** [comment]
@gkowalski I converted this PR to Draft though maybe it's already ready for review? This initial PR is becoming quite big and it's difficult to review everything. I'm OK merging it early if we can start the server manually (including documentation in README on how to do that, including the steps to follow to manually seed the DB with the i2b2 data).

**Pull request merged**